src/idl_gen_general.cpp

```diff
@@ -203,8 +203,13 @@ class GeneralGenerator : public BaseGenerator {
       const std::string &classcode, bool needs_includes) {
     if (!classcode.length()) return true;
 
-    std::string code;
-    code = code + "// " + FlatBuffersGeneratedWarning();
+    std::string code("//------------------------------------------------------------------------------\n"
```

Comment:
Right in case of the Rosly project it might be correct. But there are many other tools out there. Resharper,  Visual Studio Code, Doxygen etc. So I was thinking that the output of another Microsoft code generator (xsd.exe) will lead to the highest compatibility.
The main difference is that in the xsd output there is valid xml in the output.